cmd/flow-dps-live/main.go

```diff
@@ -176,9 +193,20 @@ func run() int {
 		loader.WithCheckpointPath(flagCheckpoint),
 	)
 
-	// TODO: Implement follower engine stuff here instead of the mock chain.
-	// FIXME: Temporary.
-	chain := mocks.BaselineChain(&testing.T{})
+	nodeID, err := flow.HexStringToIdentifier(flagNodeID)
+	if err != nil {
+		log.Error().Err(err).Msg("invalid node ID")
+		return failure
+	}
+
+	accessNodeID, err := flow.HexStringToIdentifier(flagAccessNodeID)
+	if err != nil {
+		log.Error().Err(err).Msg("invalid access node ID")
+		return failure
+	}
```

Comment:
Either `accessID`/`nodeID` (same for flags), or `ownNodeID` and `accessNodeID` (don't like as much personally).